internal/config/generate/main.go

```diff
@@ -0,0 +1,133 @@
+// (c) Cartesi and individual authors (see AUTHORS)
+// SPDX-License-Identifier: Apache-2.0 (see LICENSE)
+
+package main
+
+import (
+	"fmt"
+	"go/format"
+	"os"
+	"sort"
+	"strings"
+	"unicode"
+
+	"golang.org/x/text/cases"
+	"golang.org/x/text/language"
+
+	"github.com/BurntSushi/toml"
+)
+
+// function names
+var functions = []string{}
+
+// A value from the toml global level map.
+type EnvGet struct {
+	Default *string `toml:"default"`
+	GoType  string  `toml:"go-type"`
+	Private bool    `toml:"private"`
```

Comment:
What private means in this case? Is it related to the mapping?

Reply:
Yes, I will explain it in the docs.